Logic/igtlioSession.h

```diff
@@ -60,8 +60,10 @@ class OPENIGTLINKIO_LOGIC_EXPORT Session : public vtkObject
  
   ///
   ///  Send the given video frame from the given device. Asynchronous.
-  /*VideoDevicePointer SendFrame(std::string device_id,
-                               vtkSmartPointer<vtkImageData> image);*/
+#if defined(USE_H264) || defined(USE_VP9) || (defined(USE_X265) && defined(USE_OpenHEVC))
```

Comment:
Perhaps build some aggregate variables
OpenIGTLink_ENABLE_VIDEOSTREAMING
so that new codecs added don't have update multiple places in the code 

(I will be adding Intel hardware decoding for H264 asap).